src/http/server/handlers/static_file_handler.cr

```diff
@@ -129,6 +131,6 @@ class HTTP::StaticFileHandler
   end
 
   private def directory_listing(io, request_path, path)
-    DirectoryListing.new(request_path, path).to_s(io)
+    DirectoryListing.new(request_path.to_s, path.to_s).to_s(io)
```

Comment:
Couldn't `DirectoryListing.new` take `Path`?

Reply:
Sure. The current stdlib integrations are just intended as proof of concept and are nowhere near complete. Before I continue in this direction, the general Path API needs to be solidified and approved.